doc/source/train/examples/torch_data_prefetch_benchmark/benchmark_example.rst

```diff
@@ -0,0 +1,12 @@
+:orphan:
+
+torch_data_prefetch_benchmark
+=============================
+
+.. include:: /../../python/ray/train/examples/torch_data_prefetch_benchmark/README.rst
+    :start-line: 3
+    :end-line: 41
```

Comment:
What about just merging this section with the contents of `README.rst` and removing the `README.rst` file? The underscore notation of the page name was previously used because they were just raw python files, but with the docs it's a lot nicer with a real title!

Reply:
Thanks @matthewdeng for your suggestion! Here is the latest [doc](https://ray--22974.org.readthedocs.build/en/22974/train/examples/torch_data_prefetch_benchmark/benchmark_example.html). It looks better but the scale of the table seems too small.

Reply:
Ah, yeah unfortunately I think that's just the way our tables look like ([example](https://docs.ray.io/en/master/tune/tutorials/tune-scalability.html))